src/components/Nav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
```

Comment:
> - [ ] Based on the previous reviewer feedback,  I was unable to access your app initially because some dependencies were missing from the package.json file, kindly use the npm install command to resolve that.👍
> 
> `npm install @fortawesome/react-fontawesome`
> `npm install @fortawesome/free-solid-svg-icons`